### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by alex-nazin**
Improvements

### CODE REVIEW

File: Featureflow/FeatureflowConfig.cs [hunk 1]
```diff
@@ -8,6 +8,10 @@ public class FeatureflowConfig
         internal static readonly Uri DefaultBaseUri = new Uri("https://app.featureflow.io");
         internal static readonly Uri DefaultStreamBaseUri = new Uri("https://rtm.featureflow.io");
```
Comment (oliveroldfield) [inline Featureflow/FeatureflowConfig.cs:11]: I may have missed this in the spec, outgoing events need to go to the subdomain 'events.featureflow.io' (this means that we bypass our fastly cache when we post.

```suggestion
internal static readonly Uri DefaultEventsBaseUri = new Uri("https://events.featureflow.io");
```

File: Featureflow/RestClient.cs [hunk 2]
```diff
@@ -72,15 +72,36 @@ internal HttpClient CreateHttpClient()
...
                     _httpClient = CreateHttpClient();
+                    _httpClient.DefaultRequestHeaders.Add("Accept", "application/json");
                     throw ex;
                 }
             }
         }
 
+        internal async Task<bool> SendEventsAsync(IEnumerable<Event> events, CancellationToken cancellationToken)
+        {
+            var uri = new Uri(_config.BaseUri, FeatureflowConfig.EventsRestPath);
```
Comment (oliveroldfield) [inline Featureflow/RestClient.cs]: `_config.EventsBaseUri`

**Review by oliveroldfield — COMMENTED** [review]
Looks great, the only thing so far I can see is that we need to update the outgoing events to post to events.featureflow.io (instead of app.featureflow.io)

File: Featureflow/RestClient.cs (hunk 2 above)
Comment (alex-nazin) [reply Featureflow/RestClient.cs]: OK

**Pull request merged**